doc/man3/SSL_CTX_set_split_send_fragment.pod

```diff
@@ -1,132 +0,0 @@
-=pod
```

Comment:
Why is this file being deleted? And we don't seem to have documentation for the new functions.

Reply:
These changes  were moved and merged into another pod file, that I lost in a rebase/merge ;( 
I going to restore it !

Reply:
Doc changes restored from my backup branch ...